Makefile

```
@@ -107,6 +107,13 @@ lint: ## Run linter
 	@echo "==> Running linter"
 	golangci-lint run
 
+gen-proto:
+	buf breaking idl --against '.git#branch=main,subdir=idl'
+	buf lint idl
+	buf format idl -w
+	buf mod update idl
+	buf generate idl
```

I'm pretty sure this is what we want, but please test. In other projects I've been using a subdirectory `./idl`, but here we're putting everything directly into the root-level of the repo.

```suggestion
	buf breaking --against '.git#branch=main'
	buf lint
	buf format -w
	buf mod update
	buf generate
```